frontend/src/views/Collection/components/DatasetTab/index.tsx

```diff
@@ -40,17 +40,17 @@ const DatasetTab: FC<Props> = ({
 
   const queryClient = useQueryClient();
 
+  const invalidateCollectionQuery = useCallback(() => {
```

Comment:
move this hook before the if statement, so we don't conditionally call the hook, which is a violation